.github/workflows/run-integration-tests.yml

```diff
@@ -6,58 +6,38 @@ jobs:
   on-linux:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
+        python-version: ["3.11"]
+        agent: ["SWEAgent", "PlannerAgent", "MonologueAgent", "CodeActAgent"]
+        sandbox: ["ssh", "exec"]
         include:
-          - name: SWEAgent-py311-ssh
-            python-version: "3.11"
-            agent: "SWEAgent"
-            embedding-model: "none"
-            sandbox: "ssh"
-          - name: PlannerAgent-py311-ssh
-            python-version: "3.11"
-            agent: "PlannerAgent"
-            embedding-model: "none"
-            sandbox: "ssh"
-          - name: MonologueAgent-py311-ssh
-            python-version: "3.11"
-            agent: "MonologueAgent"
+          - agent: "MonologueAgent"
```

Comment:
Shouldn't we remove this line (and L17, L19, L21)?

Reply:
I don't quite follow.

Reply:
sorry, my bad; this looks good